sstables/sstables.hh

```diff
@@ -593,6 +593,8 @@ private:
     // It is initialized to 0 to prevent the sstables manager from reclaiming memory
     // from the components before the SSTable has been fully loaded.
     mutable std::optional<size_t> _total_reclaimable_memory{0};
+    // Total memory reclaimed so far from this sstable
+    size_t _total_memory_reclaimed{0};
```

Comment:
Don't know if we truly need to maintain this if we have a list of sstables. As long as we're under our limit, we can keep unreclaiming.